concordia/views.py

```diff
@@ -1,20 +1,34 @@
 import csv
 import os
 import sys
+import shutil
+from django.http import HttpResponse
 from logging import getLogger
-
 import requests
 from django.conf import settings
 from django.contrib.auth import get_user_model
 from django.contrib.auth.mixins import LoginRequiredMixin
 from django.contrib.auth.models import User
+from django.views.generic import TemplateView
+from registration.backends.simple.views import RegistrationView
 from django.core.paginator import Paginator
 from django.db.models import Count, Q, Sum
 from django.http import HttpResponse
 from django.shortcuts import get_object_or_404, redirect, render
 from django.urls import reverse
-from django.views.generic import TemplateView
-from registration.backends.simple.views import RegistrationView
+from django.contrib.auth.models import User
+from django.shortcuts import get_object_or_404
+from django.db import transaction
+from django.db.models import Q
+from django.db.models import Count, Sum
+
+PROJECT_DIR = os.path.dirname(os.path.abspath(__file__))
+BASE_DIR = os.path.dirname(PROJECT_DIR)
+
+sys.path.append(BASE_DIR)
```

Comment:
Tinkering with the system path is usually a mistake. It is definitely a mistake in view code or to do so unconditionally.